docs/installation/compatibility.md

```diff
@@ -0,0 +1,71 @@
+# Ensuring compatibility with legacy frameworks
+
+GOV.UK Frontend has been designed to work alongside the 'legacy frameworks' -
+[GOV.UK Frontend Toolkit], [GOV.UK Template] and [GOV.UK Elements].
+
+If you are using components from GOV.UK Frontend within a page that also
+includes any of the legacy frameworks, you should [enable compatibility
+mode](#compatibility-mode).
+
+If a part of your service uses GOV.UK Frontend without including the legacy
+frameworks, but you need to provide a consistent experience with these
+frameworks for your users, you can configure GOV.UK Frontend to [use the legacy
+colour palette](#using-the-legacy-colour-palette).
+
+Enabling either of these options means that you are no longer benefiting from
+many of the accessibility and usability improvements made in GOV.UK Frontend, as
+it prioritises providing a consistent experience for the user.
+
+In order to enable either of these options you'll need to have [installed GOV.UK
+Frontend using npm](installing-with-npm.md). These features are not available if
+you are using the 'dist' version.
+
+
+## Compatibility mode
+
+You can enable compatibility mode for each legacy framework individually.
+
+You can do this by including configuration in your application's Sass file,
+before you include GOV.UK Frontend:
+
+```SCSS
+$govuk-compatibility-govukfrontendtoolkit: true;
+$govuk-compatibility-govuktemplate: true;
+$govuk-compatibility-govukelements: true;
+
+@import "govuk-frontend/all";
+```
+
+If compatibility mode is enabled for any framework, GOV.UK Frontend will:
+
+- use the legacy colour palette
+- no longer use rem for font sizes
+- include adjustments to the CSS to 'counter' specific CSS rules in the legacy
```

Comment:
I think it might be worth mentioning how these variables affect the font that is loaded.